inference-engine/tests/functional/shared_test_classes/src/single_layer/constant.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/constant.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string ConstantLayerTest::getTestCaseName(const testing::TestParamInfo<constantParamsTuple>& obj) {
+    std::vector<size_t> data_shape;
+    InferenceEngine::Precision data_precision;
+    std::vector<std::string> data_elements;
+    std::string targetName;
+
+    std::tie(data_shape, data_precision, data_elements, targetName) = obj.param;
+
+    std::ostringstream result;
+    result << "S=" << CommonTestUtils::vec2str(data_shape) << "_";
+    result << "dataPRC=" << data_precision.name() << "_";
+    result << "dataValue=" << CommonTestUtils::vec2str(data_elements) << "_";
```

Comment:
Am I right that `data_elements` are constant values? If yes I propose to don't use all elements in the test case name (maybe only first N elements) because we can have tests which will have hundreds of elements.

Reply:
Done